assets/units/data.json

```diff
@@ -2598,7 +2598,7 @@
 	{
 		"id": 33,
 		"name": "Golden Wyrm",
-		"playable": false,
+		"playable": true,
```

Comment:
@DreadKnight Is there a mechanism for enabling creatures to test?

Reply:
Well, this is the mechanism, Golden Wyrm unit has cardboard added. If you do mean something else I'm afraid you'll have to be more specific :D

Reply:
I basically meant is it ok to enable a creature in development?

Reply:
@nandastone Oh right, it's a bit of a double edged sword atm tbh. For now hoping you will finish the unit you enabled / working on faster, but player base is low for now. Anyway, I've opened issue #1963 on how to handle this somewhat better.